### ISSUE TITLE
Configuration settings and driver for Win 3.11 with S3 VIRGE?

### ISSUE THREAD

**Issue by emendelson** [joncampbell123/dosbox-x#3362]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar question(s) already exists?

- [X] I have searched and didn't find any similar issues.

### Question

Is it now possible to run Win 3.11 with S3 Virge drivers? Using the latest code, I've tried various combinations of drivers and conf file settings, but, when Windows starts up, I only see the letter "T" at the upper left of the window, and then DOSBox-X locks up. I've tried machine=svga_s3virge and saga_s3virgevx, and I've tried installing the drivers found here:

http://vogonsdrivers.com/getfile.php?fileid=301&menustate=0

It seems from current commits that this should be possible, but I can't make it work. Is there any specific color depth and screen size that might work? If so, I haven't found it.

Thanks for any help with this.

**Comment by rderooy** [comment]
Virge support is still in development by @joncampbell123 if you want to try it, make sure your using the latest code from git.

**Comment by emendelson** [comment]
Yes, I built the code yesterday, and couldn't make it work. If anyone can suggest the settings that will eventually work, I'll continue testing every few days.

**Comment by joncampbell123** [comment]
You'll need Virge drivers and machine=svga_s3virge

**Comment by joncampbell123** [comment]
S3 ViRGE development is much farther along than it was in March, and it appears to work perfectly fine in Windows 3.1 through Windows 98. Care to try again?
